### PR TITLE
remove ad mob

### LINKED ISSUE #18: Remove google_mobile_ads from pubspec
For now, the App doesn't and souldn't contain ADS, because it is against MAWAQIT Philosophy.

The google_mobile_ads is non necessary for now. Could you please remove it from pubspec ?

  # ADS
  google_mobile_ads: ^1.0.1

### PR DESCRIPTION
**Pull request by ismaelAbogabal**
Fixes #18

### CODE REVIEW

File: lib/src/models/ad_state.dart [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
+//     //   'AdMod_ Native ad clicked: ${nativeAd.adUnitId}.',
+//     // ),
+//     // onNativeAdImpression: (nativeAd) => print(
+//     //   "AdMod_ Native ad impression: ${nativeAd.adUnitId}",
+//     // ),
+//     // onRewardedAdUserEarnedReward: (ad, reward) => print(
+//     //   'AdMod_ User rewarded: ${ad.adUnitId}, ${reward.amount} ${reward.type}.',
+//     // ),
+//   );
+// }
```
Comment (ibrahim-zehhaf-mawaqit) [inline lib/src/models/ad_state.dart:55]: please don't keep commented code, you can remove it

File: pubspec.yaml [hunk 2]
```diff
@@ -47,7 +47,7 @@ dependencies:
   flutter_inappwebview: ^5.3.2
 
   # ADS
-  google_mobile_ads: ^1.0.1
+#  google_mobile_ads: ^1.0.1
```
Comment (ibrahim-zehhaf-mawaqit) [inline pubspec.yaml:50]: remove it instead comment

Comment (ibrahim-zehhaf-mawaqit) [reply pubspec.yaml:50]: you have to do a flutter get pub to update the pubspec.lock file

Comment (ismaelAbogabal) [reply pubspec.yaml]: @ibrahim-zehhaf-mawaqit done.

**Pull request merged**